### ISSUE TITLE
Utah water right in Prod with a legal status: Rejected

### ISSUE THREAD

**Issue by amabdallah** [WSWCWaterDataExchange/MappingStatesDataToWaDE2.0#166]
Not sure what the cause of this but worth investigating 

https://westdaat.westernstateswater.org/details/right/UTwr_WR439349?state=N4IgtgziBcIQhgFwKYBtUEsUFoBuBOEAGnHgAcZQAvAexrABllc0YAOAOgHYA2AVgBM%2BAIxd8ABnwAWHmxElUSLAFcAJshgBmfBwHDxw6VOk9NghTQB2AcxXqY2YU46bxbcXx7Dj%2BX-i4AvgFAA

**Comment by rwjam** [comment]
Double checking.  Is the comment here why there are UTwr in WaDE / WestDAAT prod that have a legal status = Rejected or Withdrawn?

**Comment by amabdallah** [comment]
Yes, I assume that Prod should not have water rights with a legal status indicating "in-active" from Michelle's list. Is that right? If so, could you remove those terms whenever you can, with this upcoming data update related to POUs and PODs mapping/review?

**Comment by rwjam** [comment]
This has been addressed.  The following UTwr records are being rejected, based on Michelles's list
- Rejected, Disallowed, Withdrawn, Temp Applications, Unapproved, Forfeited, Lapsed, Expired, Terminated
